spec/spec.md

```diff
@@ -451,14 +451,32 @@ This profile currently assumes that ID Token and a single VP passed as a VP Toke
 
 Note that a Holder DID signing the ID Token in its `sub` claim is user's identifier within the RP/Verifier, while a Holder DID signing a VP in its `iss` claim is user's identifier within the Issuer, and the two do not have the same connotation.
 
-#### ID Token Validation
+#### [[def:Validation of Authorization Response]]
```

Comment:
```suggestion
#### Validation of Authorization Response
```

Reply:
def enables us to link it in other parts of the spec. lets leave this here since I reference this section in the earlier section

Reply:
I think I was trying to reference the sections without changing the section name - I was doing it wrong than..